src/lib/containers/entity_finder/details/configurations/EntityChildrenDetails.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import { useGetEntityChildrenInfinite } from '../../../../utils/hooks/SynapseAPI/useGetEntityChildren'
+import { Direction, EntityHeader, SortBy } from '../../../../utils/synapseTypes'
+import { DetailsView } from '../view/DetailsView'
+import { EntityDetailsListSharedProps } from '../EntityDetailsList'
+
+type EntityChildrenDetailsProps = EntityDetailsListSharedProps & {
+  parentContainerId: string
+}
+
+export const EntityChildrenDetails: React.FunctionComponent<EntityChildrenDetailsProps> = ({
+  sessionToken,
+  parentContainerId,
+  includeTypes,
+  showVersionSelection,
+  selectColumnType,
+  selected,
+  selectableTypes,
+  toggleSelection,
+}) => {
+  const [sortBy, setSortBy] = useState<SortBy>(SortBy.NAME)
+  const [sortDirection, setSortDirection] = useState<Direction>(Direction.ASC)
+
+  const {
+    data,
+    status,
+    isFetching,
+    hasNextPage,
+    fetchNextPage,
+  } = useGetEntityChildrenInfinite(sessionToken, {
```

Comment:
just to verify, is this container is constrained?  that is, will the infinite scroll (if it was a very long list) make it difficult to access the components below it, or is it in a container with a vertical scroll bar?

Reply:
The container for the table has a defined height. The number of items in the table won't affect the size or location of anything else, so there's no UX issue with infinite scroll here